templates/santa/application/participants_discord.html.twig

```diff
@@ -13,18 +13,25 @@
 {% endmacro %}
 
 {% block user_item %}
+    {% set user = userForm.parent.vars.choices[userForm.vars.value].data %}
+
     <label
             class="user-item"
-            for="user-{{ user.identifier }}"
+            for="{{ userForm.vars.id }}"
             data-search-index="{{ user.name ~ ' ' ~ user.extra.nickname }}"
     >
-        <input type="checkbox"
-               {{ user.identifier in selectedUsers ? 'checked="checked"' : '' }}
-               name="users[]"
-               value="{{ user.identifier }}"
-               id="user-{{ user.identifier }}"
-               data-summary="{{ current.userSummary(user)|escape('html_attr') }}"
-        >
+
+        {{ form_widget(userForm, (user.identifier in selectedUsers ?
+            {'attr': {
+                'checked': 'checked',
```

Comment:
why do you force the `checked` attribute here?

Reply:
It should be handled by Symfony directly if your submitted data are passed correctly :wink: